src/Workspaces/VisualBasic/Portable/Extensions/SemanticModelExtensions.vb

```diff
@@ -80,17 +80,20 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
 
         <Extension()>
         Public Function GenerateNameForArgument(semanticModel As SemanticModel,
-                                                argument As ArgumentSyntax) As String
-            Dim result = GenerateNameForArgumentWorker(semanticModel, argument)
+                                                argument As ArgumentSyntax,
+                                                cancellationToken As CancellationToken) As String
+            Dim result = GenerateNameForArgumentWorker(semanticModel, argument, cancellationToken)
             Return If(String.IsNullOrWhiteSpace(result), s_defaultParameterName, result)
         End Function
 
         Private Function GenerateNameForArgumentWorker(semanticModel As SemanticModel,
-                                                       argument As ArgumentSyntax) As String
+                                                       argument As ArgumentSyntax,
+                                                       cancellationToken As CancellationToken) As String
             If argument.IsNamed Then
                 Return DirectCast(argument, SimpleArgumentSyntax).NameColonEquals.Name.Identifier.ValueText
             ElseIf Not argument.IsOmitted Then
-                Return semanticModel.GenerateNameForExpression(argument.GetExpression())
+                Return semanticModel.GenerateNameForExpression(
+                    argument.GetExpression(), capitalize:=False, cancellationToken:=cancellationToken)
```

Comment:
➡️ As was explained by Cyrus in response to my other comment, the answer to my vague question is no such option current exists for parameters, so we pass `False` since that's what users are generally expecting today. :smile: